src/NexusMods.App.UI/Controls/MiniGameWidget/Standard/MiniGameWidgetViewModel.cs

```diff
@@ -28,7 +30,7 @@ public MiniGameWidgetViewModel(ILogger<MiniGameWidgetViewModel> logger,
             .WhenAnyValue(vm => vm.Game)
             .Where(game => game is not null)
             .OffUi()
-            .SelectMany(LoadImage)
+            .SelectMany(game => ImageHelper.LoadGameIconAsync(game, (int)ImageSizes.GameThumbnail.Width, _logger))
```

Comment:
We can use R3 for the async stuff

```suggestion
            .ToObservable()
            .SelectAwait((game, token) => ImageHelper.LoadGameIconAsync(game, (int)ImageSizes.GameThumbnail.Width, _logger))
            .AsSystemObservable()
```

Reply:
This will require other changes as SelectAwait requires a ValueTask. 
I can do that as a separate PR.